### ISSUE TITLE
Accessibility: create key, tools, project namespaces

### ISSUE THREAD

**Issue by lvuch** [rancher/dashboard#4391]
Master: https://github.com/rancher/dashboard/issues/4324
7.
## Shared theme

- [Repeats in other pages too] Heading order incorrect. The titles do not follow a semantic HTML order and they jump from H1 to H5.
- Text label missing from input ![](RackMultipart20211014-4-19dhorn_html_5bb2f85f6e3a5ba4.png) **image referenced** **.** Consider improving color contrast for the placeholder text.
- [Repeats in other pages] Dropdown options fail color contrast:

## Light theme

- ![](RackMultipart20211014-4-19dhorn_html_607feed89c60575f.png) **image referenced**
- ![](RackMultipart20211014-4-19dhorn_html_4295e482d00bd8b9.png) **image referenced** Button Fails color contrast. Consider the same recommendations provided above for Icons within buttons with label.

## Dark theme

- Input fails contrast &amp; label missing

![](RackMultipart20211014-4-19dhorn_html_3ef2c25d7fa896b.png) **image referenced**

8.
## Shared issues

- Same issues as in other pages: images missing alt tag and muted text fails contrast **.**![](RackMultipart20211014-4-19dhorn_html_cd8130aef4bdc9ff.png) **image referenced**

- ARIA role presentation must be removed when the element is made visible, as it is not allowed for the element

![](RackMultipart20211014-4-19dhorn_html_e35decf2b921c1f3.png) **image referenced**

9.
## Shared issues

- Star project/Namepsace needs an interactive semantic. [More on this.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#focusable_elements_should_have_interactive_semantics) ![](RackMultipart20211014-4-19dhorn_html_3efc31916ffbd5d1.png) **image referenced**
- Color contrast in muted text is not enough ![](RackMultipart20211014-4-19dhorn_html_f117ee6bc50ea572.png) **image referenced**
